libs/portals/shared-modules/delegations/src/components/delegations/DelegationViewModal.tsx

```
@@ -0,0 +1,164 @@
+  onClose,
+  isAdminView,
+  ...rest
+}: DelegationViewModalProps) => {
+  const { formatMessage, lang } = useLocale()
+  const { userInfo } = useAuth()
+  const isOutgoing = direction === 'outgoing'
+  const [getAuthScopeTree, { data: scopeTreeData, loading: scopeTreeLoading }] =
+    useAuthScopeTreeLazyQuery()
+
+  useEffect(() => {
+    if (delegation && delegation.domain) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)